tests/kubectl_test.go

```diff
@@ -156,9 +155,10 @@ var _ = Describe("[sig-compute]oc/kubectl integration", func() {
 
 		Context("'kubectl get vmim'", func() {
 			It("print the expected columns and their corresponding values", func() {
-				vmi := tests.NewRandomVMIWithEphemeralDisk(cd.ContainerDiskFor(cd.ContainerDiskCirros))
-				tests.AddUserData(vmi, "cloud-init", "#!/bin/bash\necho 'hello'\n")
-
+				vmi := libvmi.NewCirros(
+					libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+					libvmi.WithNetwork(v1.DefaultPodNetwork()),
```

Comment:
@EdDev @acardace  apparently, before we moved to `libvmi.NewCirros`, `NewRandomVMIWithEphemeralDisk` implicitly created a migratable VM with `Masquerade`. Now we force each test to "know" about it.

How about changing `libvmi.NewCirros()`  migratable by default? I think that this would make our test more representative of the typical cloud use case. On the other hand, one may argue that this change of default should take place in the production code.

WDYT?

Reply:
I think that it represents reality (the current default) and it is not clear which is used more.
I guess we can know that only after all are migrated to `libvmi`.

There is now a [helper](https://github.com/kubevirt/kubevirt/blob/86f6b7daddeef6ad704c5d190bedf3c52ec4ebf7/tests/libvmi/network.go#L46) which can make it a bit easier to consume, but it includes cloud-init and may not be a fit for Cirros.

But I do not really have any strong objection to change the factory defaults. As long as it fits most uses cases, that should be fine.
You just need to think about if these details contribute to the test understanding or not. Having defaults and hiding information is not always good.

Reply:
so I'll keep this as is?

Reply:
There are cases where we use other interface and network types for the VMIs and I think that with the current libvmi implementation if we use masquerade as a default you will always get masquerade (in a hidden way) AND the interface you really want, I think that's not ideal and therefore I would leave it as is.

Also even if there's a bit more to type to get a migratable VMI it's also really explicit about the VMI definition we're testing and I like that.